src/eventDispatchers/touchEventHandlers/addNewMeasurement.js

```diff
@@ -7,13 +7,18 @@ import {
   addToolState,
   removeToolState,
 } from '../../stateManagement/toolState.js';
+import debug from 'debug';
+
+const log = debug('cornerstoneTools')
+  .extend('eventDispatchers')
+  .extend('touchEventHandlers');
```

Comment:
The pattern I'm trying to follow with log scoping is to roughly follow the file path.
There are some cases where the scope might be shared across files within a directory.